about.html

```diff
@@ -0,0 +1,172 @@
+
+                        <p class="p4">This year, we hope to expand our invitation list including organizations and
+                            individuals who
+                            want to work with us on shared projects that advance the cause of the Commons, free culture
+                            and
+                            open knowledge.</p>
+                        <p class="p5">if you're active and engaged in the worlds of open content and knowledge - free
+                            software
+                            advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments
+                            and
+                            foundations, lawyers, and activists - we hope you'll consider joining us this year to build
+                            a
+                            stronger, more vibrant commons together.</p>
+                    </div>
+                </div>
+                <div class="con">
+                    <p>Please contact us per Email for nay further questions about CC Global Summit 2015!</p>
+                    <h3>ccsummit2015@cckorea.org</h3>
+                </div>
+            </div>
+
+        </div>
+
+
+        <div class="about-logo">
+            <div class="container">
+                <h4>Creative Commons
+                    Global Summit 2015 Logo</h4>
+                <img src="images/title_underline.svg" alt="underline">
+                <p class="p6">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7,
+                    July.</p>
+                <p class="p7">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7,
+                    July.
+                    After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won
+                    the prize.
+                    After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as
+                    follows:</p>
+                <img class="aboutLogo" src="images/about_logo 1.svg" alt="about Logo">
+            </div>
+            
+        </div>
+
+        <div class="prev-summit">
+            <div class="container">
+                <h5>See the past CC Global Summits</h5>
+            <img src="images/title_underline.svg" alt="underline">
+            <p>Take a look at the past two CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+            <div class="prev-summitImg">
+                <img src="images/2011.svg" alt="about 2011">
+                <img src="images/2013.svg" alt="about 2013">
+            </div>
+            <div class="prev-summitImg2">
+                <img src="images/about_2011 1.svg" alt="about 2011">
+                <img src="images/about_2013 1.svg" alt="about 2013">
+            </div>
+            </div>
+        </div>
+
```

Comment:
- [ ] Kindly fix the spaces.

- Behance design
<img width="533" alt="image" src="https://github.com/abdozayan12/capstone-project---Conference-page/assets/41163571/cfdde3ea-ae53-40f0-8575-cf0c5e8e4e1b">

- Your design
<img width="509" alt="image" src="https://github.com/abdozayan12/capstone-project---Conference-page/assets/41163571/4e027c40-8874-4da7-b754-141145730689">